doc/Makefile.am

```diff
@@ -1,4 +1,4 @@
-EXTRA_DIST = doxygen.cfg.in
+EXTRA_DIST = doxygen.cfg.in libusb.png
```

Comment:
On Tue 2016-Mar-29 01:16 AM KIMURA Masaru wrote:
>
> In doc/Makefile.am <https://github.com/libusb/libusb/pull/177#discussion_r57598772>:
>
> > @@ -1,4 +1,4 @@
> > -EXTRA_DIST = doxygen.cfg.in
> > +EXTRA_DIST = doxygen.cfg.in libusb.png
>
> it should not be part of solaris backend?
>

When building docs in the libusb-1.0.20.tar.bz2 distribution, there's an error like below:

error: Project logo 'libusb.png' specified by PROJECT_LOGO does not exist!

I added it to the distribution. If it's not appropriate in the Solaris backend push, I can remove it.

Thanks,
Lei Chen

> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/libusb/libusb/pull/177/files/85ec215f03229fc1e571d3a802e49ff0f40eef06#r57598772>
>